src/pages/invoices/common/hooks/useResolveInputField.tsx

```diff
@@ -16,13 +16,19 @@ import { useHandleProductChange } from './useHandleProductChange';
 import { useTranslation } from 'react-i18next';
 import { InputField } from '@invoiceninja/forms';
 import { useCurrentInvoice } from 'common/hooks/useCurrentInvoice';
-import { ChangeEvent } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 import { useHandleLineItemPropertyChange } from './useHandleLineItemPropertyChange';
 import { useFormatMoney } from './useFormatMoney';
 import { InvoiceItem } from 'common/interfaces/invoice-item';
 import { isNonNumericValue } from 'common/helpers/invoices/resolve-non-numeric-value';
+import { CurrencyInput } from 'components/forms/CurrencyInput';
+import { ClientResolver } from 'common/helpers/clients/client-resolver';
+import { CurrencyResolver } from 'common/helpers/currencies/currency-resolver';
+import { Client } from 'common/interfaces/client';
+import { useCurrentCompany } from 'common/hooks/useCurrentCompany';
+import { Currency } from 'common/interfaces/currency';
 
-const numberInputs = ['discount', 'cost', 'unit_cost', 'quantity'];
+const numberInputs = ['discount', 'unit_cost', 'quantity'];
```

Comment:
Can you share the reason why `cost` was removed?

Reply:
Initially I thought this input should be used for item cost only and i did separate check for that column but it has come to my knowledge that this input is used for all number inputs on invoice page.